dao/pom.xml

```diff
@@ -68,8 +57,5 @@
             <artifactId>log4j-core</artifactId>
             <version>2.12.1</version>
         </dependency>
-
-
     </dependencies>
-
 </project>
```

Comment:
set up your IDE to add a new line at the end of files

Reply:
@valevaty98 done